mods/mc_worldmanager/realm/realmSchematicSaveLoad.lua

```diff
@@ -94,11 +94,21 @@ end
 ---@return boolean whether the schematic fit entirely in the realm when loading.
 function Realm:Load_Schematic(schematic, config)
 
+
+    local schematicStartPos = self:LocalToWorldPosition(config.startOffset)
```

Comment:
This line is currently causing a crash when generating realms from schematics via commands: see #144 for info

Reply:
Fixed